src/main/resources/biokotlin-tools_environment.yml

```diff
@@ -0,0 +1,10 @@
+name: biokotlin-tools-conda
```

Comment:
Which commands require these?   I am not sure we need a conda env here

Reply:
@zrm22 This was in the CI for PHGv2, I can remove it if you want? This is where it's used in the CI .yml
      - name: Run Conda
        run: conda env create --solver=libmamba --file src/main/resources/biokotlin-tools_environment.yml